src/compiler/diagnosticMessages.json

```diff
@@ -5553,6 +5553,30 @@
         "category": "Message",
         "code": 95107
     },
+    "Add a return statement": {
+        "category": "Message",
+        "code": 95108
+    },
+    "Remove block body braces": {
+        "category": "Message",
+        "code": 95109
+    },
+    "Wrap this object literal with parentheses": {

```

Comment:
I think it would be more consistent to remove "this".

Reply:
Yeah, it's actually confusing in the case `const f: () => I1 = () => {a: 1};` because the squiggle and code fix are on `f`, not the object literal.